sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_models.py

```diff
@@ -215,15 +215,16 @@ def properties(self):
     def value(self):
         # type: () -> str
         """
-        The secret value
+        The secret's value
```

Comment:
Could you also help us conform to [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) with this PR? We need the first line of every docstring to follow the opening `"""`. So this one should be:
```
"""The secret's value

:rtype: str
"""
```

Reply:
ok, going through and changing that